packaging/ubuntu-16.04/control

```diff
@@ -29,6 +29,7 @@ Build-Depends: autoconf,
                libudev-dev,
                openssh-client,
                pkg-config,
+               policykit-1,
```

Comment:
Pardon my ignorance - why do we need this build-dependency? Does that install some special magic for xgettext to be able to extract the strings?

Reply:
Yes, it installs `/usr/share/gettext/its/polkit.{its,loc}` which `xgettext` needs to know how to deal with `.policy` files. An option would be to include `polkit.its` in source instead.